gymfc.ini

```diff
@@ -3,7 +3,7 @@
 # In most cases this should not be modified
 [DEFAULT]
 
-SetupFile = /usr/share/gazebo-9/setup.sh
+SetupFile = /usr/local/share/gazebo/setup.sh
```

Comment:
@wil3 instead of making this change I can find another way like symlink if that is more convenient for you. Note that this change uses Gazebo 11